backend/dev/app.ts

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = require('./router.js');
+
+const app = express();
+const port:number = 4200;
```

Comment:
it is good practice to pass the port through environment variables. (you can google what environment variables are)

good practice two is to describe the default value if the environment variable is not set

Reply:
Yeah, I have already used it in our last project. Thanks for reminding me :)